gateway-server/src/test/java/org/apache/knox/gateway/services/topology/DefaultTopologyServiceTest.java

```diff
@@ -177,6 +181,17 @@ public void testGetTopologies() throws Exception {
     }
   }
 
+  /**
+   * Set the static GatewayServices field to the specified value.
+   *
+   * @param gws A GatewayServices object, or null.
+   */
+  private void setGatewayServices(final GatewayServices gws) throws Exception {
+    Field gwsField = GatewayServer.class.getDeclaredField("services");
+    gwsField.setAccessible(true);
+    gwsField.set(null, gws);
+  }
+
```

Comment:
If I understand well this is implemented because EasyMock does not support static method mocking. I'm ok with this change, however, you may consider using [PowerMock](https://github.com/powermock/powermock)

Reply:
Your understanding is correct.